commands/publish/__tests__/publish-command.test.js

```diff
@@ -360,6 +362,33 @@ Map {
     });
   });
 
+  describe("--summary-file", () => {
+    it("skips creating the summary file", async () => {
+      const cwd = await initFixture("normal");
+      const fsSpy = jest.spyOn(fs, "writeFileSync");
+      await lernaPublish(cwd);
+
+      expect(fsSpy).not.toHaveBeenCalled();
+    });
+
+    it("creates the summary file", async () => {
```

Comment:
Please add coverage for the case where you don't pass the additional value to `--summary-file`, I don't think it's going to work right now...

Reply:
Yeah good call, I've updated